src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -282,7 +282,11 @@ public function process()
                 $bMsg .= 'ActionId: ' . $this->_lastActionId . "\r\n" . $aMsg;
                 $event = $this->_messageToEvent($bMsg);
                 $response = $this->findResponse($event);
-                $response->addEvent($event);
+                if ($response === false) {
```

Comment:
That's ok, take your time. I'm interested in knowing if this is just a bug in pami, or a strange asterisk behavior, etc.